### PR TITLE
Adds documentation on cluster shutdown and startup

### LINKED ISSUE #329: Describe Graceful Shutdown and Startup
Describe how to shut down a cluster properly
How to start a cluster back up with all intended workloads

For example for moving machines to a different datacenter, migrating to other machines, in preparation for a maintenance window or emergency, startup after an emergency and so on

### PR DESCRIPTION
**Pull request by chentex**
# Describe your changes
Describe how to shut down a cluster properly
How to start a cluster back up with all intended workloads

For example for moving machines to a different datacenter, migrating to other machines, in preparation for a maintenance window or emergency, startup after an emergency and so on

# Related Issues / Projects
Please provide links to Bugzilla and other GitHub projects with your description if they are related to the changes.

Relates to:
fixes #329

### CODE REVIEW

File: adoc/admin-shutdown-startup.adoc [hunk 1]
```diff
@@ -0,0 +1,18 @@
+[#shutdown-startup]
+= Shutdown & Startup
+
+Instructions to safely shutdown and startup a cluster.
+
+Shutting down::
+- Take a link:{docurl}/single-html/caasp-admin/#_backup[backup]
```
Comment (chentex) [inline adoc/admin-shutdown-startup.adoc:7]: I don;t know if this link works completely @r0ckarong if you can confirm.

**Comment by r0ckarong** [comment]
@chentex Could you please add the respective commands and considerations for each step? 

E.g.:

"Scale applications down to zero"
Which commands do you run, how do you find out it works, how do you check the status, what is the status when you are done?

"Wait for them to come online and are on a healthy status."
What would you run to check on this, how long would you expect this to take, what can you do to troubleshoot if this doesn't happen?

Write as if you're explaining this to someone who has never done this before (like me).

**Review by r0ckarong — CHANGES_REQUESTED** [review]
@chentex Could you please add the respective commands and considerations for each step?

E.g.:

"Scale applications down to zero"
Which commands do you run, how do you find out it works, how do you check the status, what is the status when you are done?

"Wait for them to come online and are on a healthy status."
What would you run to check on this, how long would you expect this to take, what can you do to troubleshoot if this doesn't happen?

Write as if you're explaining this to someone who has never done this before (like me).

**Comment by chentex** [comment]
Done updating @r0ckarong

**Review by r0ckarong — APPROVED** [review]

**Comment by r0ckarong** [comment]
@chentex Could you have another look please. Did formatting and wording review. Added some warnings. If you approve I can merge this.

**Comment by chentex** [comment]
I see no problems with the modifications, This is good to go.

**Pull request merged**